Tests/Integration/Components/ImportServiceTest.php

```diff
@@ -277,67 +276,48 @@ public function testFindRemoteArticleIdsByCategoryId()
 
     public function testImportRemoteCategory()
     {
-        $this->importFixtures(__DIR__ . '/../_fixtures/simple_connect_items.sql');
-
-        $bookCategoryId = 1111;
-        $this->manager->getConnection()->executeQuery(
-            'INSERT INTO s_plugin_connect_categories (id, category_key, label, shop_id) VALUES (1111, "/deutsch/bücher", "Bücher", 1234)');
-
-        $this->manager->getConnection()->executeQuery(
-            'INSERT IGNORE INTO `s_plugin_connect_product_to_categories` (`articleID`, `connect_category_id`) VALUES (?, ?)',
-            [14471, $bookCategoryId]
-        );
-
-        $this->manager->getConnection()->executeQuery(
-            'INSERT IGNORE INTO `s_plugin_connect_categories` (`category_key`, `label`, `shop_id`) VALUES (?, ?, ?)',
-            ['/deutsch/bücher/fantasy', 'Fantasy', 1234]
-        );
-        $fantasyCategoryId = $this->manager->getConnection()->lastInsertId();
-
-        $this->manager->getConnection()->executeQuery(
-            'INSERT IGNORE INTO `s_plugin_connect_product_to_categories` (`articleID`, `connect_category_id`) VALUES (?, ?)',
-            [14471, $fantasyCategoryId]
-        );
-
-        // insert invalid articleId in s_plugin_connect_product_to_categories
-        $this->manager->getConnection()->executeQuery(
-            'INSERT IGNORE INTO `s_plugin_connect_product_to_categories` (`articleID`, `connect_category_id`) VALUES (?, ?)',
-            [9087041234, $fantasyCategoryId]
-        );
+        $this->importFixtures(__DIR__ . '/_fixtures/one_item.sql');
```

Comment:
one_item sounds too generic. I think will be better to use article name or category name if the file contains only one item.